src/API/src/schema.gql

```diff
@@ -45,6 +46,21 @@ type Query {
   userRole(id: String!): UserRole!
 }
 
+"""sample data"""
+type Sample {
```

Comment:
Not sure why this was added - maybe I ran a build that hadn't been run before? Either way, we want it here anyway...

Reply:
We should try to work this out and make sure it isn't some sort of configuration specific to your machine. Not something for this PR though.